tests/functional/object/mcg/test_noobaa_priority_class.py

```
@@ -371,7 +372,14 @@ def _wait_for_pvpool_pods_priority_class(
                 continue
             all_match = True
             for pod in pods:
-                pod_dict = pod.get()
+                try:
+                    pod_dict = pod.get()
+                except CommandFailed:
+                    logger.info(
+                        f"Pod {pod.name} disappeared during reconciliation, retrying"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)